server/views/pages/liteAlreadyApproved.html

```diff
@@ -0,0 +1,34 @@
+{% set mainClasses = 'govuk-!-width-two-thirds' %}
+{% extends "../partials/layout.html" %}
+{% from "button/macro.njk" import govukButton %}
+{% from "panel/macro.njk" import govukPanel %}
+
+{% set pageTitle = "Already approved" %}
+
+{% block beforeContent %}
+
+{% include "../partials/breadCrumb.html" %}
+
+{% endblock %}
+
+{% block content %}
+
+  {{ govukPanel({
+    titleText: "Categorisation has already been approved"
+  }) }}
```

Comment:
I think the govukWarning (with that '!' icon would be better than the green panel - to show that what has happened is 'less than ideal' !